main.go

```diff
@@ -1487,6 +1487,13 @@ func proxyMapConfig() (*client.ProxyMap, bool) {
 	p := proxy.Proxy{}
 	p.GrpcMetadata = v.GetStringSlice("proxy_grpc_metadata")
 	p.HttpHeaders = v.GetStringSlice("proxy_http_headers")
+	if len(p.HttpHeaders) > 0 {
+		var normalizedHeaders []string
+		for _, header := range p.HttpHeaders {
+			normalizedHeaders = append(normalizedHeaders, strings.ToLower(header))
+		}
+		p.HttpHeaders = normalizedHeaders
```

Comment:
Nit: we can reuse the orignal slice without creating a new one.
```suggestion
		for i, header := range p.HttpHeaders {
			p.HttpHeaders[i] = strings.ToLower(header)
		}
```

Reply:
Thank you, updated.